### ISSUE TITLE
App settings screen: design support CTA

### ISSUE THREAD

**Issue by CourtneyLee333** [techmatters/terraso-product#1182]
We need to add a link to the new website for help documentation, and perhaps a CTA for feedback/engagement with us. Incorporate the new Terms of Service stuff:

- In the Settings screen, add 
- Privacy Notice (linked)
- Terms of Service (linked)
- Help and Support (linked: [landpks.terraso.org/help/help](https://landpks.terraso.org/help/help/))

### Acceptance criteria
- [x] Design settings screen updates

### Design
[Figma](https://www.figma.com/design/KMJbhjVyrrLFHj6jGcX9lN/LandPKS-Design-Prototype-working-file?node-id=11353-43201&t=VmWr5Gx67f9POVZd-1)

**Comment by DerekCaelin** [comment]
@CourtneyLee333 I'm going to try to bundle this update to the sign-in screen with an un-implemented task: 
https://github.com/techmatters/terraso-product/issues/950

Please bear in mind the location of the build number as you think about this design.

**Comment by CourtneyLee333** [comment]
@DerekCaelin Sorry, I'm confused as to why app settings screen updates are bundled with the sign-in screen task. What's the relationship?

**Comment by DerekCaelin** [comment]
You are right, the work described in this issue is unrelated to the work involving an updated sign-in screen. Please see #1184 for more details on the sign-in screen updates.
